src/o365/teams/commands/report/teams-report-useractivityusercounts.ts

```diff
@@ -0,0 +1,87 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from "../../../base/GraphCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  period: string;
+}
+
+class TeamsReportUserActivityUserCountsCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TEAMS_REPORT_USERACTIVITYUSERCOUNTS}`;
+  }
+
+  public get description(): string {
+    return 'Get the number of Microsoft Teams users by activity type';
```

Comment:
@sprider , could you please keep that text the same as in the md documents.

Reply:
This is complete @VelinGeorgiev. Please review.